### PR TITLE
test(forms-web-app): fix test title for application number redirect

### PR DESCRIPTION
**Pull request by a6software**
## Ticket Number

UCD-892

## Description of change

Fixup test title - no code change here folks

## Checklist

- [ ] Requires infrastructure changes
- [ ] I have updated the documentation accordingly
- [x] I have merged commits to avoid fixing things in this PR
- [x] New features have tests (with the right names)
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `master` (please only [rebase](https://github.com/foundry4/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

File: forms-web-app/jest.config.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ module.exports = {
   coverageThreshold: {
     global: {
       branches: 90,
-      functions: 91,
+      functions: 90,
```
Comment (a6software) [inline forms-web-app/jest.config.js:13]: this is unrelated to this PR.

A new file has been added in a previous PR which has zero coverage. I bumped up the minimums in the previous PR. This new file then takes us below the minimums. To be addressed separately.

**Review by Fenisay — APPROVED** [review]

**Comment by a6software** [comment]
ty
